clash_lib/src/proxy/hysteria2/convert.rs

```diff
@@ -0,0 +1,43 @@
+use std::sync::Arc;
```

Comment:
you can put this under clash_lib/src/proxy/converters

Reply:
ok